doc/users/interactive.rst

```diff
@@ -280,25 +270,24 @@ If ``ipympl`` is installed use the magic:
 
 to select and enable it.
 
-If you only need to use the classic notebook you can use
+If you only need to use the classic notebook, you can use
 
 .. sourcecode:: ipython
 
   %matplotlib notebook
 
-which uses the `.backend_nbagg` backend which ships with Matplotlib.
-However nbagg does not work in Jupyter Lab.
+which uses the `.backend_nbagg` backend which is installed by Matplotlib; 
```

Comment:
```suggestion
which uses the `.backend_nbagg` backend which part of Matplotlib; 
```

"is installed by" implies it is an external package that we depend on which it is a first-party backend.

Reply:
which part of -> from